examples/air/air.py

```diff
@@ -99,15 +109,6 @@ def __init__(self,
             self.cuda()
 
     def prior(self, n, **kwargs):
```

Comment:
i personally find the language choices of prior/model/model_step a bit confusing... also, the order in which the functions appear in the file. can we choose better names?

Reply:
> can we choose better names?

We can try! I'm so familiar with them that I can't see the problem. What names would you have found less confusing?

> also, the order in which the functions appear in the file

Sure. Let me know your preferred order and I'll adjust them. (In a follow up PR.)